module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2449,21 +2470,32 @@ protected function mapRenewalBlockReason($reason, $itype)
     }
 
     /**
-     * Return a location for a Koha item
+     * Return a location (branch or shelving) for a Koha item
      *
      * @param array $item Item
      *
      * @return string
      */
     protected function getItemLocationName($item)
     {
-        $libraryId = (!$this->useHomeLibrary && null !== $item['holding_library_id'])
-            ? $item['holding_library_id'] : $item['home_library_id'];
-        $name = $this->translateLocation($libraryId);
-        if ($name === $libraryId) {
-            $libraries = $this->getLibraries();
-            $name = isset($libraries[$libraryId])
+        $name = '';
+        if ($this->config['ItemTypeRenewalBlockMappings']['Location'] == 'Branch') {
```

Comment:
It would be a good idea to add a default value in case the configuration is undefined -- e.g.

```
if ($this->config['ItemTypeRenewalBlockMappings']['Location'] ?? 'Branch' == 'Branch') {
```